app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -308,14 +308,33 @@
           </div>
         {{/if}}
       {{/if}}
+      {{#if canAcceptOmise}}
+        <label>{{t 'Payment with Omise'}}</label>
+        <div class="field payments">
+          <div class="ui checkbox">
+            {{input type='checkbox' id='payment_by_omise' checked=data.event.canPayByOmise}}
+            <label for="payment_by_omise">
+              <span>{{t 'Yes, accept payment through Omise Gatewat'}}</span>
```

Comment:
why the unnecessary span tag?

Reply:
So the user may get more information about `OMISE` through redirection to their gateway

Reply:
No I meant the `<span>` tag. Span tags without classes don't serve any purpose.